Sources/Build/PkgConfig.swift

```diff
@@ -103,20 +103,49 @@ struct PkgConfigParser {
         }
     }
     
-    private func parseDependencies(_ depString: String) -> [String] {
-        let exploded = depString.characters.split(separator: " ").map(String.init)
+    /// Parses `Requires: ` string into array of dependencies.
+    /// The dependecy string has seperator which can be (multiple) space or a comma.
+    /// Additionally each there can be an optional version constaint to a dependency.
+    private func parseDependencies(_ depString: String) throws -> [String] {
         let operators = ["=", "<", ">", "<=", ">="]
-        var deps = [String]()
-        var skipNext = false
-        for depString in exploded {
-            if skipNext {
-                skipNext = false
-                continue
+        let seperators = [" ", ","]
+        
+        // Look at a char at an index if present.
+        func peek(idx: Int) -> Character? {
+            guard idx <= depString.characters.count else { return nil }
+            return depString.characters[depString.characters.startIndex.advanced(by: idx)]
+        }
+        
+        // This converts the string which can be seperated by comma or spaces
+        // into an array of string.
+        func tokenize() -> [String] {
+            var tokens = [String]()
+            var token = ""
+            for (idx, char) in depString.characters.enumerated() {
+                // Encountered a seperator, use the token.
+                if seperators.contains(String(char)) {
+                    // If next character is a space skip.
+                    if let peeked = peek(idx: idx+1) where peeked == " " { continue }
+                    // Append to array of tokens and reset token variable.
+                    tokens.append(token)
+                    token = ""
```

Comment:
I think this body could be written more simply as:
```
if separators.contains(String(char))
  if !token.isEmpty {
    tokens.append(token)
  }
} else {
  token += String(char)
}
```

FWIW, it would be more efficient (and possibly more obvious) to iterate over the indices and record the start index when a new token is encountered, then append a new token if a start index had been seen.

Also, I wonder if the pc language allows "foo>=2"? Does it require separators?